packages/terra-switch/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+ChangeLog
+=========
+
+Unreleased
+----------
+### Added
+* Added Switch Component
```

Comment:
```suggestion
* Initial release
```

Reply:
[changed to initial release](https://github.com/cerner/terra-core/pull/2846/commits/8ba4fb42d6a30245f9e12acc308252d47b22d7d2#diff-291cda9ad2bfa679e05b0bca3809bb19).